apps/src/hooks/use-url-sync.ts

```diff
@@ -174,10 +133,32 @@ export const useUrlSync = () => {
 				params.set(URL_PARAMS.DATE_RANGE, currentDateRangeStr);
 			}
 		}
+
+		// List of configurations that are simply saved as is in the params if
+		// their value is different from their default value
+		const simpleConfigurations: [keyof ClimateVariableConfigInterface, string][] = [
+			['forecastType', URL_PARAMS.FORECAST_TYPE],
+			['forecastDisplay', URL_PARAMS.FORECAST_DISPLAY],
+			['interactiveRegion', URL_PARAMS.INTERACTIVE_REGION],
+			['dataValue', URL_PARAMS.DATA_VALUE],
+			['colourScheme', URL_PARAMS.COLOUR_SCHEME],
+			['colourType', URL_PARAMS.COLOUR_TYPE],
+			['averagingType', URL_PARAMS.AVERAGING_TYPE],
+			['version', URL_PARAMS.VERSION],
+			['frequency', URL_PARAMS.FREQUENCY],
+		];
+
+		for (const [configKey, urlParam] of simpleConfigurations) {
+			const defaultValue = defaultConfig?.[configKey];
+			const currentValue = climateData[configKey];
+			if (currentValue != undefined && currentValue !== defaultValue) {
+				params.set(urlParam, currentValue.toString());
+			}
+		}
 	};
```

Comment:
(question/observation): All of this function could be moved outside, it doesn't affect anything within the scope of the closure.